setup/web_server_configuration.rst

```diff
@@ -31,6 +31,16 @@ to use PHP :ref:`with Nginx <web-server-nginx>`.
 
 .. _web-server-apache-mod-php:
 
+Using Symfony Flex in Apache
+----------------------------
+
+The recommended way is to install the apache recipe by executing the following command:
+
+.. code-block:: terminal
+
+    $ cd my-project/
+    $ composer require symfony/apache-pack
+
```

Comment:
One way or another, the user will need the rewrite rules. They can either put them in `public/.htaccess` or put them in their virtual host. So:

* Let's rename this section to something like "Adding Rewrite Rules"
* Change the text from "recommended way" to "the easiest way".
* We should also say (in the text) what this does: it will install a `public/.htaccess` file
* No need to show the `cd my-project`

Then, after the command, mention that you can get a performance boost by putting the rewrite rules in your VirtualHost and changing `AllowOverride All` to `AllowOverride None`